modules/statics/src/account.ts

```diff
@@ -183,6 +187,32 @@ export class HederaCoin extends AccountCoinToken {
   }
 }
 
+/**
+ * The Algo network supports tokens (assets)
+ * Algo tokens work similar to native ALGO coin, but the token name is determined by
+ * unique asset id on the chain. Internally, BitGo uses token identifiers of the format: (t)algo:<assetId>
+ *
+ */
+export class AlgoCoin extends AccountCoinToken {
+  public assetURL: string;
+
+  constructor(options: AlgoCoinConstructorOptions) {
+    super({
+      ...options,
+    });
+
+    if (options.assetURL) {
+      try {
+        new URL(options.assetURL);
+      } catch (ex) {
+        throw new InvalidDomainError(options.name, options.assetURL);
+      }
```

Comment:
this an overly broad catch

for instance, on node 8 there is no global `URL` so this will incorrectly throw `InvalidDomainError`

please guard this with `e.code == 'ERR_INVALID_URL'`